Dockerfile

```diff
@@ -9,11 +10,24 @@ FROM golang:alpine as builder
 # flag -ldflags "-s -w" produces a smaller executable
 RUN go install -ldflags "-s -w" -v github.com/a8m/envsubst/cmd/envsubst@v1.3.0
 
+FROM node:18-alpine3.18 as assets
+
+ARG WALLABAG_VERSION
+
+RUN set -ex \
+ && apk add --no-cache curl \
+ && curl -L -o /tmp/wallabag.tar.gz https://github.com/wallabag/wallabag/archive/$WALLABAG_VERSION.tar.gz \
+ && tar xvf /tmp/wallabag.tar.gz -C /tmp \
+ && mv /tmp/wallabag-$WALLABAG_VERSION /tmp/wallabag \
+ && cd /tmp/wallabag \
+ && yarn install \
+ && yarn build:prod
```

Comment:
I think you don't need to build them as they are already part of the artifact in the GitHub release.
Just curl the archive and extract the assets might be enough.

Reply:
as the URL used is `https://github.com/wallabag/wallabag/archive/$WALLABAG_VERSION.tar.gz` I think it's the archive of the git repo, so (soon) without the assets, that's why I build them.
if we don't want to build them here then we should use the release archive that we build, and I don't know if there is a standard URL to be used to retrieve it

Reply:
It's always `https://github.com/wallabag/wallabag/releases/download/VERSION/wallabag-VERSION.tar.gz` where `VERSION` is `2.6.6` as of today. So you might be able to retrieve the version using `WALLABAG_VERSION` I guess

Reply:
done